docs/rules/README.md

```diff
@@ -257,6 +257,7 @@ For example:
 | [vue/require-name-property](./require-name-property.md) | require a name property in Vue components |  | :hammer: |
 | [vue/script-indent](./script-indent.md) | enforce consistent indentation in `<script>` | :wrench: | :lipstick: |
 | [vue/sort-keys](./sort-keys.md) | enforce sort-keys in a manner that is compatible with order-in-components |  | :hammer: |
+| [vue/space-between-siblings](./space-between-siblings.md) | Insert newlines between sibling tags in template  | :wrench: | :warning: |
```

Comment:
I think the rule name should start with `padding-line-between-`. I don't think `space` mean just blank lines.
I also think `siblings` is ambiguous. I think `padding-line-between-tags` is probably better.